content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,113 @@
+- After that, we bill you monthly at the fixed price per collaborator or environment for your plan.
+- If at any point during the current month you remove any previously added collaborators or environments from the monthly quota, we issue a prorated refund for the unused time until the end of the month.
+
+### Quotas for the Prisma Data Proxy
+
+- Each plan includes different Data Proxy CPU usage quotas which reset at the beginning of every month.
+- The **Early Adopter** and **Free** plans have a hard limit of monthly Data Proxy CPU usage.
+- If a project reaches the hard limit on **Early Adopter** and **Free** plans, we will pause your project until the beginning of next month when the Data Proxy CPU usage quota resets.
+- We will send notifications in advance before your project reaches the hard limit, so that you have time to consider upgrading to a paid plan to avoid service interruptions.
+- On the **Scale** and **Enterprise** plans, the Data Proxy usage quota is unlimited (no hard limits). Both plans include an initial quota of Data Proxy usage and additional usage exceeding the quota is billed on a monthly basis.
+- We do not issue refunds for unused Data Proxy CPU usage quotas.
+
+## <strong>Free</strong> plan
+
+The **Free** plan is suitable as a project playground for any project before it reaches the phase of scaling to a mature Prisma application that requires more resources to scale.
+
+If you started using the Prisma Data Platform after the announcement of its General Availability on Jun 16, 2022, any Prisma projects that you created or added run on the **Free** plan.
+
+- Quota of 3 team members
+- Quota of 2 environments
+- Quota of 3,000,000 CPU ms of Data Proxy usage per project
+- 100 query engines to process your app queries
+- Supported team member roles
+  - Admin only
+
+## <strong>Scale</strong> plan
+
+When your Prisma project scales and you need to ensure a significant amount of Prisma Data Proxy usage for connection pooling and optimization, you can upgrade to the **Scale** plan.
+
+- Quota of 5 team members
+- Quota of 5 environments
+- Quota of 30,000,000 CPU ms of Data Proxy usage per project
+- 100 query engines to process your app queries
+- Supported team member roles
+  - Admin
+  - Collaborator
+  - Developer
+  - Viewer
+
+## <strong>Enterprise</strong> plan
+
+When you want to manage enterprise-level Prisma projects with an a flexible number of team members, environments, query engines, and Data Proxy usage, you can purchase the **Enterprise** plan for your projects.
+
+- Custom number of team members
+- Custom number of environments
+- Custom number of query engines
+- Custom amount of of Data Proxy usage
+- Supported team member roles
+  - Admin
+  - Collaborator
+  - Developer
+  - Viewer
+
+<Tip>
+
+To purchase the **Enterprise** plan for your projects, contact Sales.
```

Comment:
```suggestion
To purchase the **Enterprise** plan for your projects, please contact Sales.
```
We absolutely should link to the contact form here, ideally with a UTM so we know people come from docs.

Reply:
I'll be happy to as soon as I have that link.

Minor note: "please" is sometimes interpreted as a pushy request so it's mostly out of favour with Docs people.

Reply:
Nice, I didn't know about the "please" thing. Makes sense :) Feel free to discard